pages/am/3.x/breaking-changes/breaking-changes-3.15.10.adoc

```diff
@@ -0,0 +1,12 @@
+= Breaking changes in 3.15.10
+:page-sidebar: am_3_x_sidebar
+:page-permalink: am/current/am_breaking_changes_3.15.10.html
+:page-folder: am/installation-guide
+:page-layout: am
+
+
+NOTE: To take advantage of these new features and incorporate these breaking changes, **use the migration guide available link:{{ '/am/current/am_installguide_migration.html' | relative_url }}[here]**.
+
+== ResetPassword HTTP Identity Provider
+
+If you are using the HTTPIdentityProvider with the UserProvider enabled, please note that this version introduce a new method to update the user password that is different from the already existing `UpdateUser` one. So please remember to configure this endpoint in order to avoid error during password updates.
```

Comment:
```suggestion
If you are using `HTTPIdentityProvider` with `UserProvider` enabled, please note that this version introduces a new user password update method that is different from the existing `UpdateUser` one. As a result, to avoid errors during password updates, ensure that this endpoint is configured correctly.
```